src/pages/Map.js

```diff
@@ -1,25 +1,49 @@
 import React from 'react';
 import GoogleMapReact from 'google-map-react';
 import { API_KEY } from '../config.js';
+import { Icon } from '@iconify/react';
+import location12Filled from '@iconify-icons/fluent/location-12-filled';
 import './Map.css';
+import LocationPin from '../components/LocationPin.js';
 
 const location = {
-  lat: 21.694,
-  lng: 71.7979,
+  lat: 45,
+  lng: -123.456,
 };
 
 const zoomLevel = 8;
 
-export default function Map() {
+export default function Map({ locations }) {
```

Comment:
I was able to see the locations in the state in app & they are passed down as props! The icons look great! The only comment is a get a lot of warnings (12) in my console. I've tried to google it & it's a google maps API bug. It seems to do with how props are being passed down? Did y'all get these warnings too?

Reply:
@rachel-fischoff were you running it locally? I don't see console warnings when running the app from the preview deploy 🤔